### ISSUE TITLE
accessibility: repo environment incompatible with windows.

### ISSUE THREAD

**Issue by henryennis** [carbon-design-system/carbon#9219]
Working with this repo requires a linux machine as a lot of the tooling used to manage the repo are based on sh scripts & other components of the Linux OS.

Using python as the sole language for tooling and repo management would be a good solution in ensuring that contributions can come from both windows and Linux users.

Furthermore the way scss/sass is handled in the repo should be changed. Copying over scss from all of the different packages is quite inefficient: linking these files would make more sense as a single source of truth is important when developing across a large repo.

This would better adhere to IBM's accessibility spec. It doesn't really make sense that the reop is not accessible on both platforms seeing that carbons design system preaches accessibility as a core selling point.

**Comment by tay1orjones** [comment]
Hi @xSCIven, thanks for reaching out. We have a [setup guide for windows](https://github.com/carbon-design-system/carbon/blob/48deedf2fb7a343594ed844d6c852c9108a3eb53/docs/guides/setup/windows.md) that might be helpful getting the repo up and running on a windows machine.

For styles, we're currently working to migrate the setup to sass modules which should greatly increase build efficiency and clarity. Check out the [planning issue](https://github.com/carbon-design-system/carbon/issues/9162) for more info.

**Comment by xSCIven** [comment]
Hi was referring to using the styles while working in the repo, the sass modules are unrelated to the way styles are managed in the repo.

When using the subsystem on windows, the build process requires more system resources than the subsystem can offer. The build will freeze at certain points as the vm does not have access to enough memory. I have tested this with 16gb of ddr4 and it fails to build. The virtualisation environment will require tweaking before being used as by default does not have access to enough resources. In addition to this the average dev does not have more than 16gb of ram and an 8 core proc.
